packages/shared/utils/countdownCircleTimerProps.js

```diff
@@ -7,6 +7,7 @@ export const countdownCircleTimerProps = {
   children: PropTypes.oneOfType([PropTypes.func, PropTypes.node]),
   size: PropTypes.number,
   strokeWidth: PropTypes.number,
+  trailStrokeWidth: PropTypes.oneOfType([PropTypes.number, PropTypes.bool]),
```

Comment:
`PropTypes.bool` does not make much sense here, because what we gonna use it when it is set to `true`. Lets just keep it `PropTypes.number` and not required so it will have `undefined` value when it is not set.